src/main/java/br/com/sbk/sbking/networking/server/SBKingServer.java

```diff
@@ -168,7 +168,8 @@ public boolean nobodyIsConnected() {
   }
 
   public void addSpectator(UUID identifier) {
-    this.table.addSpectator(new PlayerNetworkInformation(this.identifierToPlayerMap.get(identifier)));
+    Player player = this.identifierToPlayerMap.get(identifier);
```

Comment:
Why the `identifierToPlayerMap` is a class? I checked and there is only one important method in there, that I think it could be inside the SBKingServer class. I know it is not a problem of this PR

Reply:
But as you removed some other classes as well, maybe you can consider my suggestion.

Reply:
I did this to extract the "updateNickname" responsibility out of `SBKingServer`. I just noticed it is not being called anywhere, so I will inline this map back to `SBKingServer`.